internal/service/connect/contact_flow_test.go

```diff
@@ -131,7 +131,6 @@ func testAccContactFlow_filename(t *testing.T) {
 	})
 }
 
-// Can't delete an contact flow. Test deletion of entire connect instance
 func testAccContactFlow_disappears_ConnectInstance(t *testing.T) {
```

Comment:
This name and implementation should be changed to the ContactFlow:
`testAccContactFlow_disappears`



And the on line 152, we need to test the Contact Flow resource itself, not the Connect Instance:
`acctest.CheckResourceDisappears(acctest.Provider, tfconnect.ResourceContactFlow(), resourceName),`

Reply:
@AdamTylerLynch  @GlennChia Thanks for review, made requested changes. Please check.

Reply:
@AdamTylerLynch Here is the AccTest output: 
```
$ date                                         
Mon 27 Dec 2021 22:38:27 IST
$ make testacc TESTARGS='-run=TestAccConnectContactFlow_serial' PKG_NAME=internal/service/connect
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./internal/service/connect/... -v -count 1 -parallel 20 -run=TestAccConnectContactFlow_serial -timeout 180m
=== RUN   TestAccConnectContactFlow_serial
=== RUN   TestAccConnectContactFlow_serial/disappears
=== RUN   TestAccConnectContactFlow_serial/basic
=== RUN   TestAccConnectContactFlow_serial/filename
--- PASS: TestAccConnectContactFlow_serial (552.29s)
    --- PASS: TestAccConnectContactFlow_serial/disappears (149.91s)
    --- PASS: TestAccConnectContactFlow_serial/basic (202.02s)
    --- PASS: TestAccConnectContactFlow_serial/filename (200.37s)
PASS
ok      github.com/hashicorp/terraform-provider-aws/internal/service/connect    555.793s
...
```